src/Transition/CreateNewIDFUsingRulesV9_4_0.f90

```diff
@@ -448,6 +448,26 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
 
               ! If your original object starts with E, insert the rules here
 
+             CASE('ENERGYMANAGEMENTSYSTEM:ACTUATOR')
+                  CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                  nodiff=.true.
+                  OutArgs(1:4)=InArgs(1:4)
+                  IF (MakeUPPERCase(InArgs(4)).eq.'ELECTRIC POWER LEVEL') THEN
+                    OutArgs(4)='Electricity Rate'
+                  END IF
+
+                  IF (MakeUPPERCase(InArgs(4)).eq.'GAS POWER LEVEL') THEN
+                    OutArgs(4)='NaturalGas Rate'
+                  END IF
+
+                  IF (MakeUPPERCase(InArgs(4)).eq.'ELECTRIC POWER DRAW RATE') THEN
+                    OutArgs(4)='Draw Electricity Rate'
+                  END IF
+
+                  IF (MakeUPPERCase(InArgs(4)).eq.'ELECTRIC POWER CHARGE RATE') THEN
+                    OutArgs(4)='Charge Electricity Rate'
+                  END IF
```

Comment:
If we keep these changes, the old names don't have "ELECTRIC " in them.

Reply:
OK, I reverted the above, so these lines aren't needed.  Commented them out.